src/pages/dmt5/_dhero.js

```diff
@@ -0,0 +1,317 @@
+            font-size: 32px;
+        }
+    }
+`
+const StyledHeader = styled(Header)`
+    color: var(--color-white);
+    display: flex;
+    margin-top: 0;
+    font-size: 2.4rem;
+
+    @media ${device.laptopM} {
+        font-size: 24px;
+    }
+    @media ${device.tablet} {
+        font-size: 20px;
+    }
+`
+
+const LottieWrapper = styled.div`
+    width: 100%;
+    max-width: 58rem;
+    position: absolute;
+    top: 2.3rem;
+    right: 12rem;
+
+    @media ${device.laptopM} {
+        max-width: 51rem;
+    }
+    @media ${device.tabletL} {
+        max-width: 54rem;
+        right: 3rem;
+    }
+    @media ${device.mobileL} {
+        max-width: 425px;
+        top: 0;
+        right: 0;
+        margin-bottom: 25px;
+    }
+`
+
+const LinkWrapper = styled.div`
+    display: flex;
+    margin-top: 3.2rem;
+
+    @media (max-width: 1420px) {
+        top: 480px;
+    }
+    @media ${device.laptop} {
+        top: 350px;
+        display: inline-block;
+    }
+    @media ${device.tabletL} {
+        top: 236px;
+    }
+    @media ${device.tablet} {
+        position: unset;
+        top: unset;
+        justify-content: start;
+        margin-top: 40px;
+    }
+`
+
+const StyledLinkButton = styled(LinkButton)`
+    padding: 1.4rem 1.6rem;
+    width: 100%;
+    max-width: 20.2rem;
+    font-size: 14px;
+    max-height: 4rem;
+    height: 100%;
+    margin-right: 2px;
+    border: unset;
+
+    @media ${device.tablet} {
+        padding: 1.5rem 1.6rem;
+        height: 42px;
+        white-space: nowrap;
+        display: inline-block;
+        max-width: 202px;
+        max-height: 40px;
+    }
+`
+const InformationWrapper = styled(Flex)`
+    width: 100%;
+    max-width: 56.2rem;
+
+    @media (max-width: 1376px) {
+        max-width: 44rem;
+    }
+    @media ${device.tabletL} {
+        max-width: 40rem;
+    }
+    @media ${device.tablet} {
+        top: 280px;
+        max-width: 328px;
+        padding: 0 16px;
+    }
+`
+
+const query = graphql`
+    query {
+        dbot: file(relativePath: { eq: "dbot_trade.png" }) {
+            ...fadeIn
+        }
+        dmt5: file(relativePath: { eq: "dmt5_trade.png" }) {
+            ...fadeIn
+        }
+        dtrader: file(relativePath: { eq: "dtrader_trade.png" }) {
+            ...fadeIn
+        }
+        dtrader_mobile: file(relativePath: { eq: "dtrader_trade_mobile.png" }) {
+            ...fadeIn
+        }
+        dbot_mobile: file(relativePath: { eq: "dbot_trade_mobile.png" }) {
+            ...fadeIn
+        }
+        dmt5_mobile: file(relativePath: { eq: "dmt5_trade_mobile.png" }) {
+            ...fadeIn
+        }
+    }
+`
+
+const DHero = ({
+    title,
+    background_alt,
+    background_svg,
+    background_svg2,
+    content,
+    image_name,
+    is_mobile,
+    join_us_for_free,
+    go_to_live_demo,
+    Logo,
+}) => {
+    const data = useStaticQuery(query)
+    const redirectUrl = () => {
+        const path = image_name === 'dbot' ? '/bot' : '/'
```

Comment:
why are we checking bot in dmt5 dhero?

Reply:
These codes are going to be shared between 2 other pages, but because those 2 are not ready in terms of design, so for now, I just implemented for DMT5 and did not change the other trading pages.